dev/drivers/scripts/plots/wafs/jevs_wafs_atmos_plots.sh

```diff
@@ -36,15 +36,15 @@ evs_ver_2d=$(echo $evs_ver | cut -d'.' -f1-2)
 ############################################################
 # environment variables set
 ############################################################
-export envir=dev
+export envir=prod
 
 export NET=evs
 export STEP=plots
 export COMPONENT=wafs
 export RUN=atmos
 export VERIF_CASE=grid2grid
 
-export COMIN=${COMIN:-/lfs/h2/emc/vpppg/noscrub/${USER}/${NET}/$evs_ver_2d}
+#export COMIN=${COMIN:-/lfs/h2/emc/vpppg/noscrub/${USER}/${NET}/$evs_ver_2d}
 export COMROOT=/lfs/h2/emc/ptmp/$USER
 
 export USH_DIR=$HOMEevs/ush/$COMPONENT
```

Comment:
@YaliMao-NOAA You would need to make the following changes to your driver script.

Line 1) Change to:
`#PBS -N jevs_wafs_atmos_plots`

Remove line 9.

Remove line 50.  USHevs needs to be defined in the driver script.

Line 57: Change jevs_wafs_plots to jevs_wafs_atmos_plots

Line 60. Change ptmp to stmp